plugins/org.jboss.tools.openshift.core/plugin.xml

```diff
@@ -31,10 +31,10 @@
 			version="1.0">
 			<moduleType types="jst.appclient" versions="5.0, 6.0, 7.0"/>
 			<moduleType types="wst.web" versions="1.0,1.2,1.3,1.4"/>
-			<moduleType types="jst.web" versions="2.2, 2.3, 2.4, 2.5, 3.0,3.1"/>
-			<moduleType types="jst.ejb" versions="1.0, 1.1, 2.0, 2.1, 3.0, 3.1,3.2"/>
-			<moduleType types="jst.ear" versions="1.2, 1.3, 1.4, 5.0, 6.0,7.0"/>
-			<moduleType types="jst.connector" versions="1.0, 1.5, 1.6,1.7"/>
+			<moduleType types="jst.web" versions="2.2, 2.3, 2.4, 2.5, 3.0, 3.1, 4.0"/>
+			<moduleType types="jst.ejb" versions="1.0, 1.1, 2.0, 2.1, 3.0, 3.1, 3.2"/>
```

Comment:
@jkopriva @robstryker shouldn't jst.ejb also support 4.0?

Reply:
@adietish I have not found anything about EJB 4.0, looks like last version is still 3.2